charts/internal/seed-controlplane/charts/remedy-controller-azure/templates/deployment.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: remedy-controller-azure
+  namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: controlplane
+    app: remedy-controller-azure
+spec:
+  revisionHistoryLimit: 0
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: remedy-controller-azure
+  template:
+    metadata:
+      annotations:
+        checksum/configmap-remedy-controller-azure-config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+{{- if .Values.podAnnotations }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+      labels:
+        garden.sapcloud.io/role: controlplane
+        app: remedy-controller-azure
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-public-networks: allowed
+        networking.gardener.cloud/to-private-networks: allowed
```

Comment:
I do not know the details of the remedy controller - could you quickly outline why it needs access to private ip ranges?

Reply:
The remedy controller needs to be able to talk to both API servers (Seed and Shoot), as well as the Azure API. I believe it needs then access to both public and private networks? Or am I wrong?